### ISSUE TITLE
_.debounce usage

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/john-travoltage#147]
It looks convenient, but we should add our own version of this based on view timesteps (e.g. phet-core's Timer should have debounce as a method), as debounce relies on setTimeout().

If we have playback automated tests that check DOM content (during playback of a sim), it could potentially fail, and playback of accessible features in the future would fail.

Code review for https://github.com/phetsims/john-travoltage/issues/128

**Comment by jessegreenberg** [comment]
@jobara the documentation around the _debounce calls mentions:

> we do not want to update the description for each individual electron being discharged, but rather for the operation of a single set of discharges.

> The time set here needs to be long enough that a discharge is counted as a single occurrence but short enough that multiple discharges, orchestrated by a user, do not appear to be a single one.

Would it be sufficient to update the description when the array of charges is empty? Presumably this would mean a set of discharges has completed.

**Comment by jobara** [comment]
@jessegreenberg, the flow of electrons does not have to be absolute. That is to say, that a user may move the arm away before all of the electrons have discharged. If there is an event for discharge being deactivated that could be used instead.

**Comment by jessegreenberg** [comment]
Thanks @jobara I forgot about that.  I think we can add an emitter that will fire an event whenever a charge stops.

**Comment by jessegreenberg** [comment]
I added an emitter that fires when a discharge is complete and removed the debounce usage in this sim.  The status is updated whenever the discharge stops (no more charges or hand is moved away from doorknob. @jobara does this change seem acceptable?

**Comment by jobara** [comment]
@jessegreenberg looks good. I tried it out and seems to be working great, better than before.

**Comment by samreid** [comment]
All seems complete for this issue, closing.
